frontend/src/metabase-lib/filter.ts

```diff
@@ -553,7 +553,7 @@ export function filterParts(
   );
 }
 
-function findFilterOperator(
```

Comment:
@ranquild how do you feel about exporting this function?

I feel like using opaque operators is sometimes nice for easy access to displayName, etc, but also sometimes cumbersome when we need to special-case things in the UI for individual operators (e.g. `between` filters get two inputs). 

Exposing this function makes translating the shortName to the opaque operator easier.

Reply:
LGTM